src/app/policies/views/PolicyDetailView.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div class="policy-details">
+    <LoadingBlock v-if="isLoading" />
+
+    <ErrorBlock
+      v-else-if="error !== null"
+      :error="error"
+    />
+
+    <EmptyBlock v-else-if="policy === null" />
+
+    <TabsWidget
+      v-if="policy !== null"
+      :tabs="tabs"
+    >
+      <template #tabHeader>
+        <h1
+          class="entity-heading"
+          data-testid="policy-single-entity"
+        >
+          {{ policy.name }}
+        </h1>
+      </template>
+      <template #overview>
+        <LabelList>
+          <div data-testid="policy-overview-tab">
+            <ul>
+              <template
+                v-for="(val, key) in policy"
+                :key="key"
+              >
+                <li
+                  v-if="['type', 'mesh', 'name'].includes(key)"
+                >
+                  <h4>{{ key }}</h4>
+                  <p>
+                    {{ val }}
+                  </p>
+                </li>
+              </template>
+            </ul>
+          </div>
+        </LabelList>
+        <div class="config-wrapper">
+          <YamlView
+            id="code-block-policy"
+            :content="stripTimes(policy)"
+            is-searchable
+          />
+        </div>
+      </template>
+
+      <template #affected-dpps>
+        <PolicyConnections
+          :mesh="policy.mesh"
+          :policy-name="policy.name"
+          :policy-type="props.policyPath"
+        />
+      </template>
+    </TabsWidget>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { ref } from 'vue'
+
+import PolicyConnections from '../components/PolicyConnections.vue'
+import LabelList from '@/app/common/LabelList.vue'
```

Comment:
**Issue**: You’re not importing a couple of components which leads to the page being blank in an error case (for example https://deploy-preview-732--kuma-gui.netlify.app/gui/mesh/default/policies/meshtraces/all.kuma-system).

```suggestion
import EmptyBlock from '@/app/common/EmptyBlock.vue'
import ErrorBlock from '@/app/common/ErrorBlock.vue'
import LoadingBlock from '@/app/common/LoadingBlock.vue'
import LabelList from '@/app/common/LabelList.vue'
```